package/wpe/wpeframework/Config.in

```diff
@@ -77,6 +77,7 @@ source "package/wpe/wpeframework-switchboard/Config.in"
 source "package/wpe/wpeframework-spotify/Config.in"
 source "package/wpe/wpeframework-ui/Config.in"
 source "package/wpe/wpeframework-amazon/Config.in"
+source "package/wpe/wpeframework-cobalt/Config.in"
```

Comment:
please sort A-Z (move Amazon up to the right place while you're at it). Helps finding plugins in menu config if they are properly sorted